samples/tutorials/gpu/gpu.ipynb

```diff
@@ -0,0 +1,296 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Preparation\n",
+    "\n",
+    "If you installed Kubeflow via [kfctl](https://www.kubeflow.org/docs/gke/customizing-gke/#common-customizations), you may already prepared GPU enviroment and can skip this section.\n",
+    "\n",
+    "If you installed Kubeflow Pipelines via [Google Cloud AI Platform Pipelines UI](https://console.cloud.google.com/ai-platform/pipelines/) or [Standalone manifest](https://github.com/kubeflow/pipelines/tree/master/manifests/kustomize), please follow following steps to setup GPU enviroment.\n",
+    "\n",
+    "## Add GPU nodes to your cluster\n",
+    "\n",
+    "To see which accelerators are available in each zone, run the following command or check the [document](https://cloud.google.com/compute/docs/gpus#gpus-list)\n",
+    "\n",
+    "```\n",
+    "gcloud compute accelerator-types list\n",
+    "```\n",
+    "\n",
+    "To well saving the costs, it's possible you create a zero-sized node pool for GPU and enable the autoscaling.\n",
```

Comment:
One nit comment:

Perhaps it's also worth mentioning that, it's recommended to make sure the upperbound of autoscaling fall in the range of granted quota. Otherwise very tricky condition might happen and it'll be hard to debug.

Reply:
> Thanks Renmin this is good.
> 
> One question, did you try to use `set_gpu_limit` together with `pipeline_operator_funcs` in TFX kubeflow_dag_runner? For example, [tensorflow/tfx#1539](https://github.com/tensorflow/tfx/issues/1539)
> 
> Somehow I got insufficient permission when uploading to GCS after adding that, which is weird.

I didn't try it. Prefer update TFX sample after that email thread got a conclusion on component-level GPU support in TFX. Only Trainer (and maybe TFT) step need GPU.

For your issue on GCS permission, it largely is due to the sample here is creating node-pool without specify the scope. I'm updating this PR to include the info. Thanks for the catch.

For the quota etc, I'm updating this PR to include it.